File: core/src/main/java/org/mifos/mobilewallet/core/data/paymenthub/api/BaseURL.java
package org.mifos.mobilewallet.core.data.paymenthub.api;

/**
 * Created by naman on 6/4/19.
 */

public class BaseURL {

    public static final String PROTOCOL_HTTPS = "http://";

    public static final String API_ENDPOINT = "med-connector-channel.mifos.io";
    public static final String PORT_NUMBER = "80";
    public static final String API_PATH = "/channel/";


    public String getUrl() {
        return PROTOCOL_HTTPS + API_ENDPOINT + ":" + PORT_NUMBER + API_PATH;
    }
}

File: core/src/main/java/org/mifos/mobilewallet/core/data/fineractcn/api/BaseURL.kt
package org.mifos.mobilewallet.core.data.fineractcn.api

/**
 * Created by Devansh 17/06/2020
 */
object BaseURL {

    private const val PROTOCOL_HTTPS = "http://"
    private const val API_ENDPOINT = "buffalo.mifos.io"
    private const val PORT = "4200"
    private const val API_PATH = "/api/"

    val baseURL: String
        get() = "$PROTOCOL_HTTPS$API_ENDPOINT:$PORT$API_PATH"
}